packages/expo-web-browser/ios/WebBrowserModule.swift

```diff
@@ -1,24 +1,40 @@
 // Copyright 2022-present 650 Industries. All rights reserved.
 
-import SafariServices
-import ExpoModulesCore
 import AuthenticationServices
+import ExpoModulesCore
+import SafariServices
 
 final public class WebBrowserModule: Module {
   private var currentWebBrowserSession: WebBrowserSession?
   private var currentAuthSession: WebAuthSession?
 
+  private func isValidUrl(urlString: String?) -> Bool {
```

Comment:
```suggestion
  private func isValid(url: URL) -> Bool {
    return url.scheme == "http" || url.scheme == "https"
  }
```
We can just take the URL directly here and skip the guard

Reply:
Makes sense. Done